tide_alert.module

```diff
@@ -93,3 +93,13 @@ function tide_alert_node_presave(EntityInterface $entity) {
     Cache::invalidateTags($site->getCacheTagsToInvalidate());
   }
 }
+
```

Comment:
According to [this](https://api.drupal.org/api/drupal/core!lib!Drupal!Core!Entity!entity.api.php/group/entity_crud/8.8.x) link the presave link is only called on create and update and not delete. Not sure how this worked previously except to conclude that they weren't being cached previously.